curriculum/challenges/english/25-front-end-development/lab-adjacency-list-to-matrix-converter/686fdcfe055bcda9f651dd2e.md

```diff
@@ -0,0 +1,232 @@
+---
+id: 686fdcfe055bcda9f651dd2e
+title: Build an Adjacency List to Matrix Converter
+challengeType: 27
+dashedName: build-an-adjacency-list-to-matrix-converter
+---
+
+# --description--
+
+In this lab, you will build a function that converts an adjacency list representation of a graph into an adjacency matrix. An adjacency list is a dictionary where each key represents a node, and the corresponding value is a list of nodes that the key node is connected to. An adjacency matrix is a 2D array where the entry at position `[i][j]` is 1 if there's an edge from node `i` to node `j`, and 0 otherwise.
```

Comment:
```suggestion
In this lab, you will build a function that converts an adjacency list representation of a graph into an adjacency matrix. An adjacency list is a dictionary where each key represents a node, and the corresponding value is a list of nodes that the key node is connected to. An adjacency matrix is a 2D array where the entry at position `[i][j]` is `1` if there's an edge from node `i` to node `j`, and `0` otherwise.
```